package.json

```diff
@@ -69,7 +69,7 @@
     "ts-loader": "^0.6.0",
     "tslint": "^3.15.1",
     "tslint-eslint-rules": "^3.1.0",
-    "typescript": "^2.0.2",
```

Comment:
Let's not upgrade to TS 2.1.4 since we have to make sure that the generated `.d.ts` is compatible with angular which is on alder version on TS. The easiest is to wait until angular is ready.